### DISCUSSION TITLE
Usage inside Deno Deploy playground

### DISCUSSION THREAD

**Discussion by o-az** [jsr-io/jsr#412] [Q&A] [answered]
I have a package published to jsr. The package is not available on npm so I can't use https://esm.sh. I want to use the package inside Deno Deploy playground but it's not being detected. Is there a trick for this?

**Comment by lucacasonato** [comment, marked as answer]
You can import JSR packages using a JSR specifier in Deno Deploy playgrounds, such as `jsr:@luca/cases`.

Example: https://dash.deno.com/playground/great-bison-73
